src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -405,14 +408,11 @@ private void generateSummarySpreadsheet(
     }
 
     csvPrinter.print("Inactive Ballots Total");
-    // Undervotes should not be included in the Inactive Ballots count, even though we treat them
-    // as such internally. Subtract undervotes (which are static throughout a contest) from the
-    // inactive ballot totals.
-    BigDecimal numUndervotes =
-        roundTallies.get(1).getBallotStatusTally(StatusForRound.INACTIVE_BY_UNDERVOTE);
+    BigDecimal numNoRankings =
+        roundTallies.get(1).getBallotStatusTally(StatusForRound.DID_NOT_RANK_ANY_CANDIDATES);
```

Comment:
along with https://github.com/BrightSpots/rcv/pull/857/files#diff-f37dd0767c8316149ae7da21b20e0b3b0c2394a3220ad31dc01de70425232a3eR1073 this is the crux of the change